elrond-wasm-debug/src/execute_mandos.rs

```diff
@@ -377,6 +381,42 @@ fn check_tx_output(tx_id: &str, tx_expect: &TxExpect, tx_result: &TxResult) {
 		tx_expect.status,
 		tx_result.result_status
 	);
+
+	match &tx_expect.logs {
+		CheckLogs::Star => {},
+		CheckLogs::List(expected_logs) => {
+			assert!(
+				expected_logs.len() == tx_result.result_logs.len(),
+				"Log amounts do not match. Tx id: {}. Want: {}. Have: {}",
+				tx_id,
+				expected_logs.len(),
+				tx_result.result_logs.len()
+			);
+
+			for (expected_log, actual_log) in expected_logs.iter().zip(tx_result.result_logs.iter())
+			{
+				assert!(
+					actual_log.equals(&expected_log),
+					"Logs do not match. Tx id: {}.\nWant: Address: {}, Identifier: {}, Topics: {:?}, Data: {}\nHave: Address: {}, Identifier: {}, Topics: {:?}, Data: {}",
+					tx_id,
+					verbose_hex(&expected_log.address.value),
+					vec_u8_to_string(&expected_log.identifier.value),
+					expected_log.topics.iter().map(|topic| verbose_hex(&topic.value)).collect::<String>(),
+					verbose_hex(&expected_log.data.value),
+					address_hex(&actual_log.address),
+					vec_u8_to_string(&actual_log.identifier),
+					actual_log.topics.iter().map(|topic| verbose_hex(&topic)).collect::<String>(),
+					verbose_hex(&actual_log.data),
+				);
+			}
+		},
+		CheckLogs::DefaultStar => assert!(
```

Comment:
In mandos-go unspecified logs are treated as "*", but leave it like this for now, I'll go over all mandos defaults very soon, they are not properly in sync between go and rust.